### PR TITLE
Feature/11265 upgrade terraform

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Tasks: 
[AB#11265](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11265)
[AB#11283](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11283)
[AB#11286](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11286)
Activity: [AB#7746](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7746)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ resources:
     endpoint: nhs-digital-gp-it-futures
 
 pool:
-  vmImage: 'ubuntu-latest'
+  vmImage: 'Ubuntu-20.04'
```
Comment (PhysicsJon) [inline azure-pipelines.yml:23]: Are we not already getting this version using `-latest`?

**Review by PhysicsJon — COMMENTED** [review]
Will approve pending response to the comment

File: azure-pipelines.yml (hunk 1 above)
Comment (gavinwakeley1) [reply azure-pipelines.yml:23]: Nope, latest is still Ubuntu-16.04' for now. In theory we are automatically moving to it soon, but it's already 1 month late. So I have forced this to reduce the warnings from the pipeline

**Review by PhysicsJon — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (PhysicsJon) [reply azure-pipelines.yml:23]: Fair enough

**Pull request merged**